torch_geometric/data/data.py

```diff
@@ -692,6 +714,45 @@ def num_faces(self) -> Optional[int]:
             return self.face.size(self.__cat_dim__('face', self.face))
         return None
 
+    # FeatureStore interface ###########################################
+
+    def items(self):
+        return self._store.items()
```

Comment:
Add type hint + doc-string. Why is this necessary for implementing the `FeatureStore` interface?

Reply:
I spent far too long on this; it's because `Data` is now a `MutableMapping` (inherited from `FeatureStore`), which defines an `items` method, so we now have to define `items` explicitly. I'll add a docstring to this effect; the end result is that the behavior remains the same.